components/zcash_protocol/src/consensus.rs

```diff
@@ -356,7 +356,7 @@ impl Parameters for MainNetwork {
             NetworkUpgrade::Nu5 => Some(BlockHeight(1_687_104)),
             NetworkUpgrade::Nu6 => None,
             #[cfg(zcash_unstable = "zfuture")]
-            NetworkUpgrade::ZFuture => None,
+            NetworkUpgrade::ZFuture => Some(BlockHeight(3_000_000)),
```

Comment:
For testing purposes, use https://github.com/zcash/librustzcash/blob/main/components/zcash_protocol/src/local_consensus.rs; do not make these changes here.